zerver/openapi/zulip.yaml

```diff
@@ -2924,6 +2924,323 @@ paths:
+        schema:
+          type: integer
+          nullable: true
+        example: 3
+        required: false
+      - name: authentication_methods
+        type: object
+        description: |
+          Each key-value pair in the object indicates whether the authentication
+          method is enabled on this server.
+
+          **Changes**: Deprecated in Zulip 2.1, in favor of the more expressive
+          `external_authentication_methods`.
+        required: false
+      - name: notifications_stream_id
+        in: query
+        description: |
+          ID of the stream that will notify stream creations.
+        schema:
+          type: integer
+          nullable: true
+        example: 6
+        required: false
+      - name: signup_notifications_stream_id
+        in: query
+        description: |
+          ID of the stream that will notify when new users join the realm.
+        schema:
+          type: integer
+          nullable: true
+        example: 3
+        required: false
+      - name: message_retention_days
+        in: query
+        description: |
+          Indicates for how many days should messages be retained. If messages
+          are retained forever, then message_retention_days = -1.
+        schema:
+          type: integer
+          nullable: true
+        example: -1
+        required: false
+      - name: send_welcome_emails
+        in: query
+        description: |
+          A boolean indicating whether emails introducing Zulip to new users
+          will be sent.
+        schema:
+          type: boolean
+          nullable: true
+        example: true
+        required: false
+      - name: digest_emails_enabled
+        in: query
+        description: |
+          A boolean indicating whether weekly emails to inactive users.
+        schema:
+          type: boolean
+          nullable: true
+        example: true
+        required: false
+      - name: message_content_allowed_in_email_notifications
+        in: query
+        description: |
+          A boolean indicating whether or not the message content is allowed in
+          missed messages emails.
+        schema:
+          type: boolean
+          nullable: true
+        example: true
+        required: false
+      - name: bot_creation_policy
+        in: query
+        description: |
+          An integer that defines who can add bots.
+          1 = Admins and member can add bots.
+          2 = Admins and members, but only admins can add generi bots.
+          3 = Only admins can add bots.
+        schema:
+          type: integer
+          nullable: true
+        example: 1
+        required: false
+      - name: create_stream_policy
+        in: query
+        description: |
+          An integer that defines who can create streams.
+          1 = Admins and member can create streams.
+          2 = Only admins can create streams.
+          3 = Admins and full members can create streams.
```

Comment:
Let's make these groups bulleted lists.

Reply:
Can I still let the values of each option? Personally speaking, I find it very handy when the API documentation have the values for an argument such as this one. I've committed as bulleted lists, with the values, but if it's better off, I change it.

Reply:
Yeah, just to be clear, I want these details here -- just a bullet before the number `1` (You'll see when you render this; without the bullets I think everything will flow to 1 line with that markdown input)